ui/src/components/Tracks.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { FC } from "react";
+import { Track } from "../types";
+import ExplicitOutlinedIcon from "@mui/icons-material/Explicit";
+import msToTime from "../utils/msToTime";
+import ExplicitIcon from "@mui/icons-material/Explicit";
+
+export const Tracks: FC<TrackType> = ({ track, albumImage }) => {
+  if (albumImage == "") {
```

Comment:
Why do we need this?

Reply:
I included this logic, because after fetching data from the API, some API calls include a url for track images, and some do not, so I instead passed the URL through as a parameter.